src/components/atoms/card-password/CardPasswordInput.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { forwardRef } from "react";
+import styled from "@emotion/styled";
+
+const Input = styled.input(() => ({
+  backgroundColor: '#ECEBF1',
+  height: '45px',
+  width: '43px',
+  borderRadius: '7px',
+  outline: 'none !important',
+  border: 'inherit',
+  fontSize: '21px',
+  textAlign: 'center',
+  marginRight: '7px',
+  '&:focus': {
+    boxShadow: 'none',
+  },
+  '&:disabled': {
+    backgroundColor: 'white',
+  }
+}));
+
+type Props = {
+  value?: string,
+  disabled?: boolean,
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void
+}
+
+const CardPasswordInput = forwardRef<HTMLInputElement, Props>((props, ref) => {
+  const { value, disabled, onChange } = props;
+  return(
+    <>
+      { 
+        disabled ? 
+          <Input disabled placeholder='•' />
+          : <Input id="card-password-input" autoComplete="off" type="text" onChange={onChange} value={value} placeholder='' ref={ref} />
+      }
+    </>
```

Comment:
```suggestion
  if (disabled) {
    return <Input disabled placeholder='•' />;
  }
  
  return <Input id="card-password-input" autoComplete="off" type="text" onChange={onChange} value={value} placeholder='' ref={ref} />;

```
early return을 써도 좋을 것 같으네요~ 🙂

Reply:
저는 react할 때 항상 return안에서 삼항연산자로 분기처리를 했는데 
early return써도 괜찮네요!
e2dac9600cf16ac90cd927935e5d5329e8cbbdff